lib/dor/indexers/workflow_indexer.rb

```diff
@@ -36,6 +33,14 @@ def to_solr
     attr_reader :document
     delegate :processes, to: :document
 
+    def wf_name
+      @wf_name ||= document.workflowId.first
+    end
+
+    def repo
+      document.repository.first
```

Comment:
Can you say why one of these is memoized and not the other?

Reply:
`repo` is only used one time.